image/cli/mascli/functions/gitops_utils

```diff
@@ -281,12 +281,10 @@ function unlock_git_repo() {
 function git_lock_branch_name() {
 
   LOCK_NAME=$1
-  ACCOUNT_ID=$2
```

Comment:
I'm not sure it's safe to remove the ACCOUNT_ID from the lockbranch name; the lock effectively being acquired by this is global to the config repo (i.e. gitops-envs) and that can still contain config for >1 account.id. 

Removing the region.id is safe since we now rely on cluster ids being unique across all regions of an account. Although we're not actually enforcing this anywhere. As an aside, I wonder if we want to remove the region path component from our gitops-envs structure as this would effectively enforce unique cluster ids per region?

E.g.
```
mcsp-dev
   us-east-1
      cluster-1
   us-east-2
      cluster-1
```
would cause issues (as cluster id is not unique across all regions. If we removed the region path component, we'd force* a unique cluster id like so:

```
mcsp-dev
    cluster-1-us-east
    cluster-1-us-east-2
```

(*uniqueness of cluster id isn't actually enforced, since the ID and path component name can be different at present, but it at least emphasises the point)

Reply:
Thasnks, I added the ACCOUNT_ID back into the lock so we can protect that.

For the region_id I have removed the region from the path and the same update in the gitops repo